documentation/en/draft_modulators.md

```diff
@@ -0,0 +1,370 @@
+      - [2.4.2 Ondas no periódicas](#242-ondas-no-peri%c3%b3dicas)
+- [Capitulo 2 Moduladores y demoduladores](#capitulo-2-moduladores-y-demoduladores)
+  - [2.1 Señal portadora](#21-se%c3%b1al-portadora)
+  - [2.2 Señal moduladora](#22-se%c3%b1al-moduladora)
+  - [2.3 Por que modular una señal?](#23-por-que-modular-una-se%c3%b1al)
+  - [2.4 Técnicas de modulación empleadas](#24-t%c3%a9cnicas-de-modulaci%c3%b3n-empleadas)
+  - [2.5 Tipos de moduladores](#25-tipos-de-moduladores)
+    - [2.5.1 Modulación analógica:](#251-modulaci%c3%b3n-anal%c3%b3gica)
+      - [2.5.1.1 Modulación de la amplitud.](#2511-modulaci%c3%b3n-de-la-amplitud)
+      - [2.5.1.2 Modulación de la frecuencia.](#2512-modulaci%c3%b3n-de-la-frecuencia)
+      - [2.5.1.3 Modulación de la fase.](#2513-modulaci%c3%b3n-de-la-fase)
+    - [2.5.2 Modulación digital](#252-modulaci%c3%b3n-digital)
+      - [2.5.2.1 Modulación por desplazamiento de amplitud (**ASK, Amplitude Shift Keying**)](#2521-modulaci%c3%b3n-por-desplazamiento-de-amplitud-ask-amplitude-shift-keying)
+      - [2.5.2.2 Modulación por desplazamiento de frecuencia (**FSK,Frecuency Shift Keying**)](#2522-modulaci%c3%b3n-por-desplazamiento-de-frecuencia-fskfrecuency-shift-keying)
+        - [2.5.2.2.1 Multiplexacion de 2 diferentes frecuencias.</h4>](#25221-multiplexacion-de-2-diferentes-frecuenciash4)
+        - [2.5.2.2.2 Oscilador controlado por tensión (VCO).](#25222-oscilador-controlado-por-tensi%c3%b3n-vco)
+      - [2.5.2.3 Modulación por desplazamiento de frecuencia Gausiana (GFSK)](#2523-modulaci%c3%b3n-por-desplazamiento-de-frecuencia-gausiana-gfsk)
+      - [2.5.2.4 Modulación por desplazamiento de fase (**PSK, Phase Shift Keying**)](#2524-modulaci%c3%b3n-por-desplazamiento-de-fase-psk-phase-shift-keying)
+- [Capitulo 3 Filtros](#capitulo-3-filtros)
+  - [3.1 ¿Qué es un filtro de frecuencia?](#31-%c2%bfqu%c3%a9-es-un-filtro-de-frecuencia)
+  - [3.2 Clasificación de los filtros](#32-clasificaci%c3%b3n-de-los-filtros)
+    - [3.2.1 Respuesta en frecuencia](#321-respuesta-en-frecuencia)
+- [Capitulo 4 Demodulador FSK](#capitulo-4-demodulador-fsk)
+
+
+
+<h1>Modulators and signals theory</h1>
+
+Within this chapter we intend to demostrate the principles and fundamental concepts that make possible the wireless interconnection of the nodes, irradiating signals and receiving it, by applying techniques described in this section.
+
+# Chapter 1: Waves
+
+A wave consist in the propationg of disturbance of some property in the space, like:
+
+- Density.
+- Pressure.
+- Electric field.
+- Magnetic field.
+
+Implying an energy transport without matter transport. The disturbed space can contain matter (air, water, etc) or not.
+
+
+<img src="imple_pic/wave.svg" alt="drawing" height="300" width="400" align="block"/>
+
+The example in the figure generating a wave with a cord, it can be appreciate the deformity of the medium in which its represented the energy transport without having to transport the matter itself.
```

Comment:
```suggestion
The example in the figure transverse waves generation on a rope, it can be appreciate the deformity of the medium in which its represented the energy transport without having to transport the matter itself.
```